app.py

```diff
@@ -61,4 +54,12 @@ def _fk_pragma_on_connect(dbapi_con, con_record):  # noqa
     return flask_app
 
 
+def enable_sqlite_fk_constraints(flask_app):
```

Comment:
could we stick a very basic docstring on this now it's in it's own function? Just so it's clear what it does. As not 100% obvious at first. The comment out of the old code would probably suffice.

Reply:
Maybe with a slight addition about what the intended use is? (is it for unit testing? reading the PR content below)

Reply:
yeah good shout 👍